installing/installing_openstack/installing-openstack-installer-sr-iov.adoc

```diff
@@ -0,0 +1,66 @@
+[id="installing-openstack-installer-sr-iov"]
+= Installing a cluster on OpenStack that supports SR-IOV-connected compute machines
+include::modules/common-attributes.adoc[]
+:context: installing-openstack-installer-sr-iov
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on {rh-openstack-first} that can use compute machines with single-root I/O virtualization (SR-IOV) technology.
+
+== Prerequisites
+
+* Review details about the
+xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
+processes.
+** Verify that {product-title} {product-version} is compatible with your {rh-openstack} version by using the "Supported platforms for OpenShift clusters" section. You can also compare platform support across different versions by viewing the link:https://access.redhat.com/articles/4679401[{product-title} on {rh-openstack} support matrix].
+// Statement seems somewhat obsolete, though not sure about assembly placement.
+* Verify that your network configuration does not rely on a provider network. Provider networks are not supported.
+
+* Have a storage service installed in {rh-openstack}, like block storage (Cinder) or object storage (Swift). Object storage is the recommended storage technology for {product-title} registry cluster deployment. For more information, see xref:../../scalability_and_performance/optimizing-storage.adoc#optimizing-storage[Optimizing storage].
```

Comment:
You might be able to retain the "Verify..." parallelism that was started.  Maybe:

* Verify that your {rh-openstack} has a storage service installed, such as block...
* Verify that your {rh-openstack} has the metadata service enabled.

Just a thought, up to you.